frontend/src/utils/color.ts

```diff
@@ -0,0 +1,16 @@
+// Copied from https://stackoverflow.com/a/59387478
+export const lighten = (color: string, lighten: number) => {
+  const c = color.replace('#', '');
+  const rgb = [
+    parseInt(c.substr(0, 2), 16),
+    parseInt(c.substr(2, 2), 16),
+    parseInt(c.substr(4, 2), 16),
+  ];
+  let returnstatement = '#';
+  rgb.forEach((color) => {
+    returnstatement += Math.round(
+      (255 - color) * (1 - Math.pow(Math.E, -lighten)) + color
+    ).toString(16);
+  });
+  return returnstatement;
```

Comment:
The `returnstatement` is not really a statement, but rather an expression. We can make it more explicit by describing what kind of thing it represents.

```suggestion
  let lightenColor = '#';
  rgb.forEach((color) => {
    lightenColor += Math.round(
      (255 - color) * (1 - Math.pow(Math.E, -lighten)) + color
    ).toString(16);
  });
  return lightenColor;
```

Reply:
Done in 4325c03ea6a76d404b7153c711f827cc3bd626d4.